CaloCluster/fcl/prolog_trigger.fcl

```diff
@@ -26,47 +38,13 @@ CaloTrigger : {
 
 
 CaloClusterFast :
-{ 
-    module_type           : CaloClusterFast
-    caloDigiModuleLabel   : CaloDigiMaker
-    digiSampling          : @local::HitMakerDigiSampling
-    blindTime             : @local::HitMakerBlindTime
-    endTimeBuffer         : 80
-    windowPeak            : 2
-    minAmplitude          : 20
-    extendSecond          : true
-    minSeedAmplitude      : 200
-    minEnergy             : 50
-    timeCorrection        : 20.0
-    adcToEnergy           : 0.0461333
-    diagLevel             : 0
-    includeCrystalHits    : true
-}
-
-CaloClusterOnline  :
 {
-  module_type            : CaloClusterOnline
-  caloCrystalModuleLabel : CaloHitMaker
-  minClusterEnergy       : 50
-  EminSeed               : 20 
-  ExpandCut              : 1 
-  EnoiseCut              : 1
-  timeCut                : @local::HitMakerBlindTime   
-  deltaTime              : 5.01
-  diagLevel              : 0
-}  
-
-
+   producers : {CaloClusterFast : @local::CaloClusterFast}   
```

Comment:
I think that here something confusing happened. `CaloClusterFast` is the module label you defined at the very top. The various module instances should be collected under the `CaloClusterTrigger` namespace.

Reply:
I think I fixed it, please check